tests/unit/stream_alert_rule_processor/test_threat_intel.py

```diff
@@ -247,6 +247,57 @@ def test_extract_ioc_from_record(self):
             assert_equal((results[3].value, results[3].ioc_type),
                          ('abcdef0123456789abcdef0123456789', 'md5'))
 
+    def test_extract_ioc_from_record_with_private_ip(self):
+        """Threat Intel - Test extrac values from a record based on normalized keys"""
+        records = [
```

Comment:
I'd add a public IP example in here

Reply:
It considers `1.1.1.2` as public address 😄  So it is already covered by old test cases. 
The private subnet range falls into three classes.
1. class A: 10.0.0.0 – 10.255.255.255
2. class B: 172.16.0.0 – 172.31.255.255
3. class C: 192.168.0.0 – 192.168.255.255

Reply:
and a malformed IP address

Reply:
and a test log with both a public and a private ip address, ensuring that a single private entry doesn't result in the other one not being processed